admin-dev/themes/new-theme/js/pages/product/index.ts

```diff
@@ -39,4 +40,33 @@ $(() => {
   grid.addExtension(new window.prestashop.component.GridExtensions.SubmitBulkActionExtension());
   grid.addExtension(new window.prestashop.component.GridExtensions.BulkActionCheckboxExtension());
   grid.addExtension(new window.prestashop.component.GridExtensions.FiltersSubmitButtonEnablerExtension());
+  grid.addExtension(new window.prestashop.component.GridExtensions.PositionExtension(grid));
+
+  /*
+   * Tree behavior: collapse/expand system and radio button change event.
+   */
+  categoryTree('div#product_catalog_category_tree_filter');
+
+  $('#product_catalog_category_tree_filter_reset').on('click', () => {
+    categoryTree('#product_categories', 'unselect');
+    $('form#product_filter_form input[name="product[id_category]"]').val('');
```

Comment:
There's no need to specify an element if you use an ID, and, what's more important we use selectors map almost everywhere these days